src/interop/passes.jl

```diff
@@ -91,10 +91,10 @@ function Base.string(options::JuliaPipelinePassOptions)
     cleanup = options.cleanup ? "cleanup" : "no_cleanup"
     warn_missed_transformations = options.warn_missed_transformations ? "warn_missed_transformations" : "no_warn_missed_transformations"
     opt_string = "<$optlevel;$lower_intrinsics;$dump_native;$external_use;$llvm_only"
-    # if VERSION >= v"1.12.0-DEV.1007" #TODO: This might not be the correct version + 1.11 check needed
+    if VERSION >= v"1.12.0-DEV.1029" || VERSION >= v"1.11.0-rc3"
```

Comment:
` >= v"1.11.0-rc3"` matches everything > `v1.12`, so you need a more fine grained version check.